ajax/libs/react-sortable-hoc/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "react-sortable-hoc",
+  "filename": "react-sortable-hoc.min.js",
+  "version": "0.6.8",
+  "description": "Set of higher-order components to turn any list into a sortable, touch-friendly, animated list",
+  "author": {
+    "name": "Clauderic Demers",
+    "email": "me@ced.io"
+  },
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/clauderic/react-sortable-hoc.git"
+  },
+  "keywords": [
+    "react",
+    "reactjs",
+    "react-component",
+    "sortable",
+    "sortable-list",
+    "list",
+    "sortable list",
+    "smooth",
+    "animated",
+    "hoc",
+    "higher-order",
+    "component"
+  ],
+  "homepage": "https://clauderic.github.io/react-sortable-hoc/",
+  "npmName": "react-sortable-hoc",
+  "npmFileMap": [
+    {
+      "basePath": "dist/umd",
+      "files": [
+        "react-sortable-hoc*"
```

Comment:
We use excluded mode at `dist` which is as the basePath, also `**/*` can grab all the files we need.
The current pattern is included mode.
Many thanks

Ref: https://github.com/cdnjs/cdnjs/blob/master/ajax/libs/folktale/package.json#L26